classes/tax/TaxRulesGroup.php

```diff
@@ -132,24 +132,46 @@ public function getIdTaxRuleGroupFromHistorizedId($id_tax_rule)
         );
     }
 
-    public static function getTaxRulesGroups($only_active = true)
+    public static function getTaxRulesGroups($only_active = true, bool $includeRates = false)
```

Comment:
This is a breaking change

Reply:
This is not the first time we change Core classes like this 🤔

Is it an acceptable breaking change? As the method can still be used the same way with the same behaviour, or should we add new method `getTaxRulesGroupsForOptionsWithRate` ?

Reply:
@eternoendless I changed this modification so it doesn't impact the functions prototypes, using an intermediate private function

The only change now is that the returned array from `getTaxRulesGroupsForOptions` now contains an extra `rate` field, but I wouldn't consider this a breaking change 🤔 (by the way it was not used in the core)